sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/async_challenge_auth_policy.py

```diff
@@ -69,6 +71,15 @@ async def on_challenge(self, request: "PipelineRequest", response: "PipelineResp
         except ValueError:
             return False
 
+        if self._verify_challenge_resource:
```

Comment:
Most of our pipeline configuration parameters can be supplied either via Client constructor, or per-operation kwargs. Looks like this one is only the former - is that intentional?
It might make sense in this context, I don't think there's a particularly strong case for enabling/disabling this on a per operation basis.

Reply:
I honestly don't know if per-operation verification was considered -- @heaths might know if it was. I would think that it's something we could add later if it's requested

Reply:
It has to be per-client. Challenge-based auth policies should be caching per-scope bearer tokens and sending them per-call until they need to be refreshed.